src/gc/unix/cgroup.cpp

```diff
@@ -119,10 +119,11 @@ class CGroup
             return true;
         }
         
-        cpu_count = quota / period;
-        if (cpu_count < UINT32_MAX)
+        cpu_count = (double) quota / period;
+        if (cpu_count < UINT32_MAX - 1)
```

Comment:
Should this be '<=' otherwise if cpu_count is equal to (UINT32_MAX - 1) then this function will return UINT32_MAX instead of (UINT32_MAX - 1), right?
In general, I agree with @tmds here that rounding up before the 'if' statement might make the code simple. Would something like this work or I missed some details?
```
// Calculate cpu count based on quota and round it up
cpu_count = (double) quota / period  + 0.999999999;
*val = (cpu_count < UINT32_MAX) ? (uint32_t)cpu_count : UINT32_MAX;
```

Reply:
It's ok to return `UINT32_MAX`. I didn't adopt @tmds approach mostly to reduce code change noise.